src/main/charts/bamboo/values.yaml

```diff
@@ -340,7 +340,94 @@ ingress:
 # Bamboo configuration 
 #
 bamboo:
-  
+
+  # -- Supply the base URL for the installation.
+  baseUrl:
+
+  # -- Override the server/agent broker URL; this is optional.
+  brokerUrl:
+
+  # The security token with which the agent will authenticate to the Bamboo server
```

Comment:
Do we need to include details on the expected format of the token? Does it need to be of a certain length/format/encoding?